uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html

```diff
@@ -218,9 +237,6 @@ <h4 ng-show="document.assignee_contact_id.length">Assignees:</h4>
     <button type="button" class="btn btn-default" ng-click="cancel()">
       Cancel
     </button>
-    <button type="submit" ng-click="openNew = true;" class="btn btn-default" ng-disabled="!documentForm.$valid">
-      Save And New
-    </button>
     <button type="submit" class="btn btn-default btn-primary">
```

Comment:
Right!, we don't need "Save and New Button" as per the requirement,  but it sounds sensible to disable a save button if the form is invalid. Will fix it.